src/System.Drawing.Common/src/System/Drawing/Imaging/Metafile.NotSerializable.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Same as Bitmap. Metafile isn't implementing ISerializable directly: http://referencesource.microsoft.com/#System.Drawing/commonui/System/Drawing/Advanced/Metafile.cs,36. Remove this file.